### ISSUE TITLE
Dependency conflicts prevent useful output

### ISSUE THREAD

**Issue by chneukirchen** [voidlinux/xbps#113]
E.g. `xbps-install -Su` (potentially with `-n`) shows a conflict and exits, but does not show which packages would be updated, making it hard to steer around the conflict.

Perhaps the order of these outputs could be changed?

**Comment by Gottox** [comment]
There's `xbps-install -vSu`, but it's a little overkill in some situations.

**Comment by xtraeme** [comment]
This has been improved for 0.48:

```
 * libxbps: improved messages when reverse dependencies in a transaction
   are not satistfied.

   After
   -----
   $ xbps-install -un
   re2-2015.07.01_1 (update) breaks installed pkg `chromium-44.0.2403.155_1'
   Transaction aborted due to unresolved dependencies.
   $

   Before
   ------
   $ xbps-install -un
   chromium-44.0.2403.155_1 broken, needs `re2-2015.07.01_1'
   Transaction aborted due to unresolved dependencies.
   $
```

**Comment by xtraeme** [comment]
Fixed for 0.48.
